.github/workflows/conformance.yml

```diff
@@ -7,6 +7,9 @@ on:
       antrea-version:
         description: The Antrea version to test. It could be a SHA-1 value, a branch, or a tag (e.g. a7b012b, release-1.12, v1.12.0). The main branch will be used if empty.
         required: false
+      antrea-values:
```

Comment:
maybe `antrea-chart-values`?

Reply:
I don’t feel antrea-values is ambiguous in the context.